caffe2/core/blob_serialization.cc

```diff
@@ -379,7 +379,7 @@ void TensorSerializer::Serialize(
     case TensorProto_DataType_STRING: {
       proto.mutable_string_data()->Reserve(chunkSize);
       const string* content = input.template data<string>();
-      for (int i = chunkBegin; i < chunkBegin + chunkSize; ++i) {
+      for (size_t i = chunkBegin; i < chunkBegin + chunkSize; ++i) {
```

Comment:
Why not `for(const auto i : c10::irange(chunkBegin, chunkBegin + chunkSize)` ?

Reply:
Thanks, don't realized caffe2 can also reference c10 util, will modify to use irange for the rest.